mfkdf2/src/definitions/uniffi_types.rs

```diff
@@ -19,11 +21,10 @@ uniffi::custom_type!(HmacSha1Response, Vec<u8>, {
   }
 });
 
-// TODO (@lonerapier): check if mfkdf2error can be converted to anyhow error
 uniffi::custom_type!(Uuid, String, {
   remote,
   lower: |v| v.to_string(),
-  try_lift: |s: String| Uuid::parse_str(&s).map_err(uniffi::deps::anyhow::Error::msg),
+  try_lift: |s: String| Uuid::parse_str(&s).map_err(|_| MFKDF2Error::InvalidUuid).map_err(Into::into),
```

Comment:
```suggestion
  try_lift: |s: String| Uuid::parse_str(&s).map_err(MFKDF2Error::InvalidUuid),
```

Wouldn't this work if `MFKDF2Error::InvalidUuid` implements the proper `From`?